.docs/user-guide/storage-providers.md

```diff
@@ -707,3 +707,107 @@ libstorage:
   in place, it may be possible for a client to attach a volume that is already
   attached to another node. Mounting and writing to such a volume could lead to
   data corruption.
+
+## GCEPD
+
+The Google Compute Engine (GCE) Persistent Disk driver registers a driver named
```

Comment:
Hi @codenrhoden,

This driver abbreviation still reflects the old `gce` and not the updated `gcepd`. I'd add a single acronym after the _Persistent Disk_ as (GCEPD). Thoughts?